### ISSUE TITLE
ECMPS 508 Testing - Color Contrast Issues

### ISSUE THREAD

**Issue by vishnunavuluri** [US-EPA-CAMD/PI-1#1977]
Required Fixes:

RF - 1a
Skip to Content has low contrast
Update to white against blue
![image.png](https://images.zenhubusercontent.com/610014e44c513c0bad9f811c/cc28d606-95e8-451c-82bc-4351313063b8)

RF-3: Color Contrast is Insufficient (Resources Screen - Update link colors to Blue Base 005EA2)
from the Report: https://app.zenhub.com/files/287570343/f8bad95b-8fdc-4603-ab9b-8ff281ee270c/download
**Updated Prototype (Resources):** https://xd.adobe.com/view/f0838f89-2fd7-4b1f-85fb-929854ff6a16-a5d0/

https://beacon.cvpcorp.com/display/PRAC/Developer

**Comment by aprematta** [comment]
RF 1a (skip to content) passes, RF 1b is still an issue, RF-3 passes

**Comment by vishnunavuluri** [comment]
@aprematta Moved the pending item to [#2515](https://app.zenhub.com/workspaces/emissioners-scrum-board-5f36cd263511ad001a777f8e/issues/us-epa-camd/easey-ui/2515)
